src/ElementData.h

```diff
@@ -0,0 +1,53 @@
+//
+//  ElementData.h
+//  drafter
+//
+//  Created by Jiri Kratochvil on 08/08/17.
+//  Copyright (c) 2017 Apiary Inc. All rights reserved.
+//
+#ifndef DRAFTER_ELEMENTDATA_H
+#define DRAFTER_ELEMENTDATA_H
+
+#include "RefractSourceMap.h"
+
+namespace drafter
+{
+
+    template <typename T>
+    struct ElementData {
```

Comment:
If I understand this structure correctly, it holds a set of types and values that are compile-time dependent on a template parameter, `T`. Why are they encapsulated in a structure instead of a namespace?

Reply:
It collect data related to element which does come from snowcrash and are lately used to create appropriate element with required meta/attributes